changelog.d/pa-3185.added

```diff
@@ -0,0 +1,3 @@
+Tainting has better support for if statements. In particular, for
```

Comment:
A small example could be nice.

Reply:
Also I don't think this is a fix in taint analysis, but a fix in naming. It will benefit more than taint analysis.

Reply:
But you can still use a taint example to illustrate the benefit of the naming fix of course.

Reply:
Done.